x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/helpers.tsx

```diff
@@ -118,15 +118,21 @@ export const getHumanizedDuration = (from: string, interval: string): string =>
   const intervalValue = dateMath.parse(`now-${interval}`) ?? moment();
 
   const fromDuration = moment.duration(intervalValue.diff(fromValue));
-  const fromHumanize = `${Math.floor(fromDuration.asHours())}h`;
 
-  if (fromDuration.asSeconds() < 60) {
-    return `${Math.floor(fromDuration.asSeconds())}s`;
-  } else if (fromDuration.asMinutes() < 60) {
-    return `${Math.floor(fromDuration.asMinutes())}m`;
+  // Basing calculations off floored seconds count as moment durations weren't precise
```

Comment:
I stopped using the moment duration in these calculations because they weren't consistent/precise and made the tests flaky.